week-4/implement/is-prime.test.js

```diff
@@ -1,3 +1,27 @@
 // Given a positive integer num,
 // When the isPrime function is called with num as input,
 // Then it should return a boolean representing whether the num is prime
+function isPrime(num) {
+    
+    if (num <= 1) {
+        return `${num} is not prime`;
+    }
+
+    for (let i = 2; i < num; i++) {
+        if (num % i === 0) {
+            return `${num} is not prime`;
```

Comment:
Great use of short-circuiting here @Sabella-8! If we know a number is definitely prime, we can stop the loop and return, without any more computation.